expression/builtin_miscellaneous.go

```diff
@@ -445,17 +445,17 @@ func (b *builtinInetAtonSig) evalInt(row chunk.Row) (int64, bool, error) {
 			digit := uint64(c - '0')
 			byteResult = byteResult*10 + digit
 			if byteResult > 255 {
```

Comment:
The implementation does not handle `SELECT INET6_ATON('1.0002.3.4')` which returns `NULL` in MySQL.